### ISSUE TITLE
dreambooth: error: the following arguments are required: --project-name`

### ISSUE THREAD

**Issue by AlexanderKozhevin** [huggingface/autotrain-advanced#256]
usage: autotrain <command> [<args>] dreambooth
       [-h]
       --model
       MODEL
       [--revision REVISION]
       [--tokenizer TOKENIZER]
       --image-path
       IMAGE_PATH
       [--class-image-path CLASS_IMAGE_PATH]
       --prompt
       PROMPT
       [--class-prompt CLASS_PROMPT]
       [--num-class-images NUM_CLASS_IMAGES]
       [--class-labels-conditioning CLASS_LABELS_CONDITIONING]
       [--prior-preservation]
       [--prior-loss-weight PRIOR_LOSS_WEIGHT]
       --project-name
       PROJECT_NAME
       [--seed SEED]
       --resolution
       RESOLUTION
       [--center-crop]
       [--train-text-encoder]
       [--batch-size BATCH_SIZE]
       [--sample-batch-size SAMPLE_BATCH_SIZE]
       [--epochs EPOCHS]
       [--num-steps NUM_STEPS]
       [--checkpointing-steps CHECKPOINTING_STEPS]
       [--resume-from-checkpoint RESUME_FROM_CHECKPOINT]
       [--gradient-accumulation GRADIENT_ACCUMULATION]
       [--gradient-checkpointing]
       [--lr LR]
       [--scale-lr]
       [--scheduler SCHEDULER]
       [--warmup-steps WARMUP_STEPS]
       [--num-cycles NUM_CYCLES]
       [--lr-power LR_POWER]
       [--dataloader-num-workers DATALOADER_NUM_WORKERS]
       [--use-8bit-adam]
       [--adam-beta1 ADAM_BETA1]
       [--adam-beta2 ADAM_BETA2]
       [--adam-weight-decay ADAM_WEIGHT_DECAY]
       [--adam-epsilon ADAM_EPSILON]
       [--max-grad-norm MAX_GRAD_NORM]
       [--allow-tf32]
       [--prior-generation-precision PRIOR_GENERATION_PRECISION]
       [--local-rank LOCAL_RANK]
       [--xformers]
       [--pre-compute-text-embeddings]
       [--tokenizer-max-length TOKENIZER_MAX_LENGTH]
       [--text-encoder-use-attention-mask]
       [--rank RANK]
       [--xl]
       [--fp16]
       [--bf16]
       [--token TOKEN]
       [--repo-id REPO_ID]
       [--push-to-hub]
       [--validation-prompt VALIDATION_PROMPT]
       [--num-validation-images NUM_VALIDATION_IMAGES]
       [--validation-epochs VALIDATION_EPOCHS]
       [--checkpoints-total-limit CHECKPOINTS_TOTAL_LIMIT]
       [--validation-images VALIDATION_IMAGES]
       [--logging]
       [--username USERNAME]
autotrain <command> [<args>] dreambooth: error: the following arguments are required: --project-name

```
!autotrain dreambooth \
--model ${MODEL_NAME} \
--output ${PROJECT_NAME} \
--image-path images/ \
--prompt "${PROMPT}" \
--resolution ${RESOLUTION} \
--batch-size ${BATCH_SIZE} \
--num-steps ${NUM_STEPS} \
--gradient-accumulation ${GRADIENT_ACCUMULATION} \
--lr ${LEARNING_RATE} \
$( [[ "$USE_FP16" == "True" ]] && echo "--fp16" ) \
$( [[ "$USE_XFORMERS" == "True" ]] && echo "--xformers" ) \
$( [[ "$TRAIN_TEXT_ENCODER" == "True" ]] && echo "--train-text-encoder" ) \
$( [[ "$USE_8BIT_ADAM" == "True" ]] && echo "--use-8bit-adam" ) \
$( [[ "$GRADIENT_CHECKPOINTING" == "True" ]] && echo "--gradient-checkpointing" ) \
$( [[ "$PUSH_TO_HUB" == "True" ]] && echo "--push-to-hub --hub-token ${HF_TOKEN} --hub-model-id ${REPO_ID}" )
```

**Comment by AlexanderKozhevin** [comment]
New format
```
!autotrain dreambooth \
--model ${MODEL_NAME} \
--project-name ${PROJECT_NAME} \
--image-path images/ \
--prompt "${PROMPT}" \
--resolution ${RESOLUTION} \
--batch-size ${BATCH_SIZE} \
--num-steps ${NUM_STEPS} \
--gradient-accumulation ${GRADIENT_ACCUMULATION} \
--lr ${LEARNING_RATE} \
$( [[ "$USE_FP16" == "True" ]] && echo "--fp16" ) \
$( [[ "$USE_XFORMERS" == "True" ]] && echo "--xformers" ) \
$( [[ "$TRAIN_TEXT_ENCODER" == "True" ]] && echo "--train-text-encoder" ) \
$( [[ "$USE_8BIT_ADAM" == "True" ]] && echo "--use-8bit-adam" ) \
$( [[ "$GRADIENT_CHECKPOINTING" == "True" ]] && echo "--gradient-checkpointing" ) \
$( [[ "$PUSH_TO_HUB" == "True" ]] && echo "--push-to-hub --hub-token ${HF_TOKEN} --hub-model-id ${REPO_ID}" )
```
